src/sql/workbench/browser/modelComponents/divContainer.component.ts

```diff
@@ -131,7 +140,7 @@ export default class DivContainer extends ContainerBase<azdata.DivItemLayout> im
 	}
 
 	public get clickable(): boolean {
-		return this.getPropertyOrDefault<azdata.DivContainerProperties, boolean>((props) => props.clickable, false);
+		return this.getPropertyOrDefault<azdata.DivContainerProperties, boolean>((props) => props.clickable, true);
```

Comment:
why do we have to change the default value here?

Reply:
Oops, I was testing changing that. Thanks.